examples/all-clusters-app/linux/BUILD.gn

```diff
@@ -53,30 +47,46 @@ source_set("chip-all-clusters-common") {
 
 if (is_libfuzzer) {
   executable("chip-all-clusters-app-fuzzing") {
-    sources = [ "fuzzing-main.cpp" ]
+    sources = [
+      "${chip_root}/examples/lock-app/linux/src/LockEndpoint.cpp",
+      "${chip_root}/examples/lock-app/linux/src/LockManager.cpp",
+      "${chip_root}/examples/lock-app/linux/src/ZCLDoorLockCallbacks.cpp",
```

Comment:
Can we put these into a separate sourceset that both the fuzzing and non-fuzzing mains include, so we don't end up copy/pasting these across those two cases?

Reply:
Good suggestion. Fixed it in the latest commit.